pkg/operator/provisioner.go

```diff
@@ -201,3 +207,19 @@ func processMonAddresses(monAddresses []string) []string {
 	}
 	return monAddrs
 }
+
+func createImageName(pvName string) string {
+	// generate a UUID for our image name
+	u := string(uuid.NewUUID())
+
+	// image name should be under 100 chars to support kernels older than 4.7
+	// when the RBD kernel module converts the image name to an OID, it will use "rbd_id.<imageName>",
+	// so we have to leave room for that "rbd_id." prefix too.
+	pvNameMaxLen := imageNameMaxLen - len(rbdIDPrefix) - len(imageNamePrefix) - len(u) - 2 // 2 hyphens
+	if len(pvName) > pvNameMaxLen {
+		// the PV name is too long, truncate it before including it in the final image name
+		pvName = pvName[:pvNameMaxLen]
+	}
+
+	return fmt.Sprintf("%s-%s-%s", imageNamePrefix, pvName, u)
```

Comment:
some naming questions:
1) why do we need a prefix? 
2) should we include `rook` in the name?
3) can we include the cluster name in the pv name?
4) if we include those, can we get rid of the trailing uuid?